pkg/webhooks/regularuser/regularuser.go

```diff
@@ -304,10 +305,22 @@ func (s *RegularuserWebhook) SyncSetLabelSelector() metav1.LabelSelector {
 
 // NewWebhook creates a new webhook
 func NewWebhook() *RegularuserWebhook {
+
 	scheme := runtime.NewScheme()
-	admissionv1.AddToScheme(scheme)
-	corev1.AddToScheme(scheme)
-	networkv1.AddToScheme(scheme)
+	err := admissionv1.AddToScheme(scheme)
+	if err != nil {
+		logx.Fatal(err)
```

Comment:
Why `logx` over the existing `logf` package that's already imported?

Reply:
done